ci/android.yml

```diff
@@ -1,21 +1,23 @@
-name: Android CI
```

Comment:
Could you double-check the line endings here?

It looks like this is reverting the change from https://github.com/actions/starter-workflows/pull/428/files

Reply:
Should be fixed now

Reply:
I'm still seeing a large PR diff. 

Could you run `git add --renormalize ci/android.yml` and commit those changes?

Reply:
Alright give me a few minutes

Reply:
It seems good now. My fault using the github editor to resolve conflicts earlier. That messed up the line endings.